addons/stock_account/security/ir.model.access.csv

```diff
@@ -2,4 +2,5 @@ id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_account_account_stock_manager,account.account stock manager,account.model_account_account,stock.group_stock_manager,1,0,0,0
 access_stock_picking_invoicing_payments,stock.picking,stock.model_stock_picking,account.group_account_invoice,1,1,1,0
 access_stock_move_invoicing_payments,stock.move,model_stock_move,account.group_account_invoice,1,1,1,0
-access_stock_valuation_layer,access_stock_valuation_layer,model_stock_valuation_layer,stock.group_stock_manager,1,1,1,0
+access_stock_valuation_layer_user,access_stock_valuation_layer,model_stock_valuation_layer,stock.group_stock_user,1,0,0,0
+access_stock_valuation_layer_manager,access_stock_valuation_layer,model_stock_valuation_layer,stock.group_stock_manager,1,1,1,0
```

Comment:
We should not change the xmlid in stable

Reply:
I checked and it is not used in our code, so it was for better clarity but I will keep the existing one